rancher2/schema_cluster.go

```diff
@@ -312,11 +313,20 @@ func clusterFieldsV0() map[string]*schema.Schema {
 			Type:          schema.TypeList,
 			MaxItems:      1,
 			Optional:      true,
-			ConflictsWith: []string{"aks_config", "eks_config", "k3s_config", "rke_config"},
+			ConflictsWith: []string{"aks_config", "eks_config", "k3s_config", "oke_config", "rke_config"},
 			Elem: &schema.Resource{
 				Schema: clusterGKEConfigFields(),
 			},
 		},
+		"oke_config": {
+			Type:          schema.TypeList,
+			MaxItems:      1,
+			Optional:      true,
+			ConflictsWith: []string{"aks_config", "eks_config", "gke_config", "k3s_config", "rke_config"},
+			Elem: &schema.Resource{
+				Schema: clusterOKEConfigFields(),
+			},
+		},
 		"default_project_id": {
 			Type:     schema.TypeString,
 			Computed: true,
```

Comment:
`clusterFieldsV0()` refers to old cluster schema. Could you please remove `oke_config` added here?? Just needed on current schema referred by `clusterFields()`

Reply:
No problem. Done.

Reply:
You need to remove all modifications done at `clusterFieldsV0()` function,  including modified `ConflictsWith` lines

Reply:
Ah, thanks. Done.